src/main/java/com/bookstore/service/BookServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.bookstore.service;
+
+import com.bookstore.dto.BookDto;
+import com.bookstore.dto.CreateBookRequestDto;
+import com.bookstore.entity.Book;
+import com.bookstore.mapper.BookMapper;
+import com.bookstore.repository.BookRepository;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class BookServiceImpl implements BookService {
+    private final BookRepository bookRepository;
+    private final BookMapper bookMapper;
+
+    @Override
+    public BookDto save(CreateBookRequestDto createBookRequestDto) {
+        Book book = bookMapper.toBookEntity(createBookRequestDto);
+        book = bookRepository.save(book);
+        return bookMapper.toBookDto(book);
+    }
+
+    @Override
+    public BookDto getById(Long id) {
+        Book book = bookRepository.getById(id);
```

Comment:
```suggestion
        Book book = bookRepository.findById(id).orElseThrow();
```

You should use findById and return Optional